SIL.Scripture.Tests/VerseRefTests.cs

```diff
@@ -2067,6 +2067,38 @@ public void UnBridge()
 			Assert.AreEqual(new VerseRef("EXO 6:4"), new VerseRef("EXO 6:4-10").UnBridge());
 			Assert.AreEqual(new VerseRef("EXO 6:150monkeys"), new VerseRef("EXO 6:150monkeys").UnBridge());
 		}
+
+		/// <summary>
+		/// Tests the Verse property's set method with numerals from various Unicode-supported scripts
+		/// </summary>
+		[Test]
+		public void SetVerse_InterpretUnicodeNumerals()
+		{
+			VerseRef vref = new VerseRef("EXO 6:5");
```

Comment:
IMO it would be better to write this as multiple tests with one test case per test, or use the TestCase attribute. Something like:

```
[TestCase("५", TestName="Devanagari numeral")]
[TestCase("૧૬", TestName="Gujarati numeral")]
...
public void SetVerse_InterpretUnicodeNumerals(string verseNumber)
{
var vref = new VerseRef("EXO 6:5");
vref.Verse = verseNumber;
Assert.That(vref.VerseNum, Is.EqualTo(5));
}
```

Reply:
I agree. Done.